src/config/dynamic-theme-fixes.config

```diff
@@ -6863,6 +6863,12 @@ input[type=text] {
 .bq-results-table-header-cell {
     background: var(--darkreader-neutral-background) !important;
 }
+.monaco-editor .cursor,
+.monaco-editor .cursor-block,
+.monaco-editor .cursor-line {
+    background-color: #ffffff !important;
+    border-color: #ffffff !important;
```

Comment:
We can use Dark or Light scheme in Dark Reader.
This color looks nice in Dark scheme but I think it will be not visible if we change the scheming to light.

To fix this we need to use `${#000000}` instead of `#ffffff`.

This will give `#000000` black in light scheming and inverted `#ffffff` in dark.